### DISCUSSION TITLE
[Suggestion] Ability to export/import custom items

### DISCUSSION THREAD

**Discussion by vico93** [destruc7i0n/crafting#56] [Ideas]
Self-explanatory

**Comment by FaultyFunctions** [comment]
This already exists on the site: 
![image](https://github.com/destruc7i0n/crafting/assets/6508045/a61a1dc3-0bed-4ed0-b8d3-30ed6ecbb307)

**Comment by vico93** [comment]
Yeah but i was talking about export (and import) the items added by that button to a file for backup purposes

**Comment by FaultyFunctions** [comment]
Oh okay that makes sense! Thanks for clarifying

**Comment by mydkong** [comment]
I'm actually exporting it by saving some localStorage entries. I thought for a system and i was wondering if i may contribute. May I?

**Comment by mydkong** [comment]
wait it is not the issue i wanted sorry 😭

**Comment by mydkong** [comment]
i had no access to my github so i didn't see these messages im so sorry
and yes sorry
